examples/directivities/simulate_binaural_recording.py

```diff
@@ -0,0 +1,164 @@
+# 2023 Robin Scheibler
+"""
+This scripts simulates a binaural room impulse response
+
+The directivity patterns are loaded from SOFA files.
+The SOFA format is described at https://www.sofaconventions.org
+
+Get more SOFA files at https://www.sofaconventions.org/mediawiki/index.php/Files
+"""
+import argparse
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import signal
+from scipy.fft import fft, fftfreq
+from scipy.io import wavfile
+from scipy.signal import fftconvolve
+
+import pyroomacoustics as pra
+from pyroomacoustics.directivities import MeasuredDirectivityFile, Rotation3D
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Simulate a binaural room impulse response"
+    )
+    parser.add_argument(
+        "--hrtf",
+        type=Path,
+        default="mit_kemar_normal_pinna.sofa",
+        help="Path to HRTF file",
+    )
+    parser.add_argument(
+        "--source",
+        type=Path,
+        default=Path(__file__).parents[1] / "input_samples/cmu_arctic_us_axb_a0004.wav",
+        help="Path to speech or other source audio file",
+    )
+    parser.add_argument(
+        "--output",
+        type=Path,
+        default=Path(__file__).parents[1]
+        / "output_samples/simulate_binaural_recording.wav",
+        help="Path to output file",
+    )
+    parser.add_argument(
+        "--interp-order",
+        type=int,
+        default=12,
+        help="Maximum order to use in the spherical harmonics interpolation. "
+        "Setting to -1 will disable interpolation",
+    )
+    parser.add_argument(
+        "--interp-n-points",
+        type=int,
+        default=1000,
+        help="Maximum order to use in the spherical harmonics interpolation",
+    )
+    args = parser.parse_args()
+
+    interp_order = 12
+    interp_n_points = 1000
+
+    fs, speech = wavfile.read(args.source)
+    speech = speech * (0.95 / abs(speech).max())
```

Comment:
Are we normalizing speech signals here? Curious to know why?

Reply:
They may have been provided in integer format or have arbitrary scale. It seems sensible to normalize them.